src/test/java/org/jboss/xavier/integrations/EndToEndTest.java

```diff
@@ -430,11 +430,18 @@ public void configure() {
             softly.assertThat(workloadInventoryReport.getBody().getContent().stream().filter(e -> e.getComplexity().contains("Unsupported")).count()).isEqualTo(1);
             softly.assertThat(workloadInventoryReport.getBody().getContent().stream().flatMap(e -> e.getRecommendedTargetsIMS().stream()).distinct().count()).isEqualTo(4);
             softly.assertThat(workloadInventoryReport.getBody().getContent().stream().filter(e -> e.getRecommendedTargetsIMS().contains("OSP")).count()).isEqualTo(11);
+            softly.assertThat(workloadInventoryReport.getBody().getContent().stream().filter(e -> e.getRecommendedTargetsIMS().contains("RHEL")).count()).isEqualTo(4);
```

Comment:
```suggestion
            softly.assertThat(workloadInventoryReport.getBody().getContent().stream().filter(e -> e.getRecommendedTargetsIMS().contains("RHEL")).count()).isEqualTo(4);
            softly.assertThat(workloadInventoryReport.getBody().getContent().stream().filter(e -> e.getRecommendedTargetsIMS().contains("None")).count()).isEqualTo(1);
```

We could add a check for target "None" here ( I debugged into a test run to check how many were in there).

Reply:
The ` assertThat(workloadInventoryReport.getBody().getContent().toArray())` check below tests everything and hence also that there's just [one VM with `None` target](https://github.com/project-xavier/xavier-integration/pull/86/files#diff-e013e18be0a878695d95297e44f564f2R356).
But, since testing is never wrong and "better safe than sorry" always applies, i'll merge your check.
Thanks @m-brophy :+1: